pkg/util/util.go

```diff
@@ -180,3 +181,38 @@ func ClusterTLSSecretName(tcName, component string) string {
 func TiDBClientTLSSecretName(tcName string) string {
 	return fmt.Sprintf("%s-tidb-client-secret", tcName)
 }
+
+// SortEnvByName sorts env list by name.
+func SortEnvByName(env []corev1.EnvVar) []corev1.EnvVar {
+	sort.Sort(sortEnvByName(env))
+	return env
+}
```

Comment:
There existed the same usage util in `pkg/monitor/monitor/util.EnvListWrapper`, I think we could remove it and use this instead.